commands/sdcli_go_integration

```diff
@@ -27,4 +27,6 @@ if [[ -f "${PWD}/main.go" ]]; then
     PKGS="$(go list ./... | sed 1d | paste -sd "," -)"
 fi
 go test -v -tags=integration -cover -coverpkg="${PKGS}" -coverprofile=.coverage/integration.cover.out ./tests
+_EXIT_CODE=$?
```

Comment:
Why not exit at this point?

Reply:
I'm open to bailing here. If we let it continue then it will still generate a coverage report for the tests that passed which is why I started this way.

Reply:
That sounds fine to me, I have no strong feelings either way.